src/constants.js

```diff
@@ -44,7 +39,12 @@ export const CATEGORIES = [
     name: 'model',
     displayName: 'Model',
     displayFields: ['id','synonyms'],
-  }
+  },
```

Comment:
Order suggested in Slack #search by Leyla (and confirmed by Stacia) was to simply add dataset to the end of the existing list. So: 1. Gene 2. Gene Ontology 3. Disease 4. Allele 5. Model 6. HTP Dataset Index. I edited the file to match that.